lib/sync-UpdatesModel_mongo.js

```diff
@@ -0,0 +1,41 @@
+var syncUtil = require('./sync-util');
+var MongoClient = require('mongodb').MongoClient;
+var fhdb;
+var mongo;
+
+var setFHDB = function(db) {
+  fhdb = db;
+};
+
+var setConnectionUrl = function(url) {
+  MongoClient.connect(url, function(err, db) {
+    if (err) throw err;
+    syncUtil.doLog(syncUtil.SYNC_LOGGER, 'info', 'sync-UpdatesModel MongoClient connected');
+
+    mongo = db;
+  });
+};
+
+var create = function(dataset_id, data, cb) {
+  return mongo.collection(dataset_id + "-updates").insertOne(data, cb);
+};
+
+var list = function(dataset_id, eq, cb) {
+  return mongo.collection(dataset_id + "-updates").find(eq).toArray(cb);
+};
+
+var remove = function(dataset_id, data, cb) {
+  var toDelete = {
+    cuid: data.cuid,
+    hash: data.hash
+  };
+  return mongo.collection(dataset_id + "-updates").deleteOne(toDelete, cb);
+};
+
+module.exports = {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)